### PR TITLE
refactor: configloader -> configmanager

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <tao.hu@merico.dev>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description

We've changed `config loader` to `config manager` in the docs, e.g.: https://docs.devstream.io/en/release-0.8/core-concepts/core-concepts/

So I refactored the code too.

## Related Issues

N/A

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

**Review by iyear — APPROVED** [review]

**Review by IronCore864 — APPROVED** [review]

File: internal/pkg/configmanager/configmanager.go [hunk 1]
```diff
@@ -18,31 +18,86 @@ var (
...
+	}
+
+	return errors
+}
+
+type Manager struct {
+	ConfigFile string
+}
+
+func NewManager(ConfigFileName string) *Manager {
```
Comment (aFlyBird0) [inline internal/pkg/configmanager/configmanager.go:77]: Maybe unexported var `configFileName` is more suitable.

**Review by aFlyBird0 — CHANGES_REQUESTED** [review]

**Comment by aFlyBird0** [comment]
More OOP, cool!

File: internal/pkg/configmanager/configmanager.go (hunk 1 above)
Comment (aFlyBird0) [reply internal/pkg/configmanager/configmanager.go:77]: You could also fix it in next pr.

**Review by aFlyBird0 — APPROVED** [review]

File: internal/pkg/configmanager/configmanager.go (hunk 1 above)
Comment (daniel-hutao) [reply internal/pkg/configmanager/configmanager.go:77]: right! I'll fix it in no time.

**Pull request merged**